pkg/controller/controller.go

```diff
@@ -1072,114 +1085,115 @@ func (c *Controller) syncHandler(key string) error {
 
 		}
 
+		if apimVolDefined && dashVolDefined && workerVolDefined {
 
 
```

Comment:
Shall we remove the additional lines if this file is not an auto-generated one?

Reply:
These have been removed in the latest code.